tests/queries/0_stateless/02883_named_collections_override.sql

```diff
@@ -0,0 +1,79 @@
+DROP NAMED COLLECTION IF EXISTS u1;
```

Comment:
let's modify the test to be `.sh` test and use `$CLICKHOUSE_TEST_UNIQUE_NAME` as a part of collection name, it will fix `ClickHouse Stateless Tests Flaky Check` (this check runs the same test in parallel many times).

Another solution here is to just add `-- Tags: no-parallel` on top of this test, but we try to avoid that whenever possible because `no-parallel` tests slow down our CI. 

Also to will be better to define collection names with this ugly `$CLICKHOUSE_TEST_UNIQUE_NAME` at the top of the test to make test more readable, e.g.
```
u1=$CLICKHOUSE_TEST_UNIQUE_NAME_collection1
u2=$CLICKHOUSE_TEST_UNIQUE_NAME_collection2
u3=$CLICKHOUSE_TEST_UNIQUE_NAME_collection3

${CLICKHOUSE_CLIENT}  -nm --query "
<copy paste of your current .sql test using $u1, $u2, $u3 as collection names>
"
```